vendor/ember-cli/test-support-suffix.js

```diff
@@ -0,0 +1,14 @@
+/*
+  used to determine if the application should be booted immediately when `app-name.js` is evaluated
+  when `runningTests` the `app-name.js` file will **not** import the applications `app/app.js` and
+  call `Application.create(...)` on it. Additionally, applications can opt-out of this behavior by
+  setting `autoRun` to `false` in their `ember-cli-build.js`
+*/
+runningTests = true;
+
+/*
+  This file overrides a file built into ember-cli's build pipeline and prevents
+  this built-in `Testem.hookIntoTestFramework` invocation:
+
+  https://github.com/ember-cli/ember-cli/blob/v3.20.0/lib/broccoli/test-support-suffix.js#L3-L5
+*/
```

Comment:
The reason this override is needed, is because the built-in ember-cli invocation of `Testem.hookIntoTestFramework()` happens _during_ `test-support.js` evaluation, which (when we are leveraging `Embroider` or `ember-auto-import`) `QUnit` is not present (since its not eagerly evaluated, we only are guaranteed it is present when we import it). As such the `Testem.hookIntoTestFramework` invocation has been moved into our `addon-test-support/index.js` which is only ran once `QUnit` is actually defined/available.